account_invoice_overdue_reminder/migrations/13.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 Akretion France (http://www.akretion.com/)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# Copied and adapted from the OCA module intrastat_product
+
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if openupgrade.table_exists(env.cr, "account_invoice"):
+        openupgrade.logged_query(
+            env.cr,
+            """
+            UPDATE account_invoice_overdue_reminder aior
+            SET invoice_id = account_invoice.move_id
+            FROM account_invoice
+            WHERE aior.%(old_inv_id)s = account_invoice.id"""
```

Comment:
```suggestion
            UPDATE account_invoice_overdue_reminder aior
            SET invoice_id = am.id
            FROM account_move am
            WHERE aior.%(old_inv_id)s = am.old_invoice_id"""
```
if you are using OpenUpgrade project, this is better